client/my-sites/sidebar/index.jsx

```diff
@@ -156,48 +154,25 @@ export class MySitesSidebar extends Component {
 	}
 
 	trackCustomerHomeClick = () => {
-		this.trackMenuItemClick( this.props.isCustomerHomeEnabled ? 'customer-home' : 'checklist' );
+		this.trackMenuItemClick( 'customer-home' );
 		this.onNavigate();
 	};
 
 	customerHome() {
-		const {
-			canUserUseChecklistMenu,
-			canUserUseCustomerHome,
-			path,
-			siteSuffix,
-			siteId,
-			translate,
-			isCustomerHomeEnabled,
-		} = this.props;
-
-		// This will be eventually removed when Customer Home is finally live
-		const canUserViewChecklistOrCustomerHome = isCustomerHomeEnabled
-			? canUserUseCustomerHome
-			: canUserUseChecklistMenu;
-
-		if ( ! siteId || ! canUserViewChecklistOrCustomerHome ) {
+		const { canUserUseCustomerHome, path, siteSuffix, siteId, translate } = this.props;
+
+		if ( ! siteId || ! canUserUseCustomerHome ) {
 			return null;
 		}
 
-		const itemProps = isCustomerHomeEnabled
-			? {
-					label: translate( 'My Home' ),
-					selected: itemLinkMatches( [ '/home' ], path ),
-					link: '/home' + siteSuffix,
-			  }
-			: {
-					label: translate( 'Checklist' ),
-					selected: itemLinkMatches( [ '/checklist' ], path ),
-					link: '/checklist' + siteSuffix,
-			  };
-
```

Comment:
I think I know the answer, but going to ask to document here:

Is it worth preserving the Checklist item in the sidebar for sites that were created too early to qualify for the Customer Home? (Or will the checklist not be very useful, since those are older sites?)

Reply:
We made the date cut off for customer home, the same as for the checklist menu item, which was the 6th August. Under this change, any site after that date eligible for the menu item will be switched to customer home. This may grab more people than we want!